### DISCUSSION TITLE
Add Multiple Jenkins

### DISCUSSION THREAD

**Discussion by mohammad-eghlima** [MCMicS/jenkins-control-plugin#573] [Ideas]
Hi,

I am not sure if I am wrong or this feature has not implemented yet but It would be great if we could configure multiple Jenkins server.
In the server page I do not see any button allowing me to add more servers. 

It would be also nice to be able to change the name of the server in the view panel. Right now it does not allow me to change the name of the server.

![image](https://github.com/MCMicS/jenkins-control-plugin/assets/9913004/6c1b05cc-5a2c-41a4-8a9e-69511747f7ca)

**Comment by MCMicS** [comment]
Hi this is currently in development and scheduled for next major release

Se #423

For the name, it is currently not possible but I read the description set on server. So if you have set it on server it should be shown 
But with multi server support it will be maybe also implemented to set a display name
